docs/docs/playground.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-id: prototype-an-assistant
```

Comment:
If we change this we should probably add a redirect since we've posted this link places already

Reply:
https://github.com/RasaHQ/rasa-website/pull/842